components/collector/src/base_collectors/collector.py

```diff
@@ -23,9 +25,14 @@
 class Collector:
     """Collect measurements for all metrics."""
 
-    def __init__(self) -> None:
+    MAX_SLEEP_DURATION = int(os.environ.get("COLLECTOR_SLEEP_DURATION", 20))
+    MEASUREMENT_LIMIT = int(os.environ.get("COLLECTOR_MEASUREMENT_LIMIT", 30))
+    MEASUREMENT_FREQUENCY = int(os.environ.get("COLLECTOR_MEASUREMENT_FREQUENCY", 15 * 60))
```

Comment:
These should be imported from components/collector/src/base_collectors/config.py